### PR TITLE
Frontend Dashboard

### PR DESCRIPTION
**Pull request by benekex2**

### CODE REVIEW

**Comment by krzysztofwolski** [comment]
- [ ] hiding sidemenu makes breadcrumbs move
- [ ] after uploading image by dropping its container is too big (after page reload everythings fine)
- [ ] card with table and card with form have different top paddings (check shipping methods and settings view)

**Comment by weronka** [comment]
Some small things that can be fixed:
- [ ] add light gray background to clickable table rows on hover on home page
- [ ] use propper checkboxes on product details:
check
- [ ] use shadow on hover on primary action buttons
image
- [ ] prevent jumping breadcrumbs and content when toggle side navigation

File: saleor/static/js/dashboard.js [hunk 1]
```diff
@@ -206,6 +200,11 @@ function initSelects() {
   $('select:not(.browser-default):not([multiple])').material_select();
   $('select[multiple]:not(.browser-default)').select2({width: '100%'});
 }
+// Clickable rows in dashboard tables
+$(document).on('click', 'tr[data-action-go]>td:not(.ignore-link)', function(){
```
Comment (weronka) [inline saleor/static/js/dashboard.js]: space missing after function()

**Comment by weronka** [comment]
Some issues on mobile devices and edge browser:

- [ ] filter form in orders list should stay if after applying filter there is no result

edge:
- [ ] header bar is broken
- [ ] price form in product view looks strange

mobile:
- [ ] hide 'DROP IMAGES HERE TO UPLOAD' on mobile
- [ ] product list view is too wide
- [ ] customers list on android - when tap to email in table row it sends to email client
- [ ] side nav looks broken - 'release: dev' appears in strange place
- [ ] search form - icon is not vertically aligned
- [ ] product form is too wide on windows phone
- [ ] need to check is text wrap using well on breadcrumbs bar - sometimes text is too short

**Pull request merged**